components/ProjectDetails.js

```diff
@@ -3,8 +3,13 @@ import Image from "next/image";
 import ModalDelete from "@/components/ModalDelete";
 import { useState } from "react";
 import router from "next/router";
```

Comment:
```suggestion
import { useRouter } from "next/router";
```
The router should be implemented via the useRouter hook. Inside the component function you only have to add `const router = useRouter();`